rewriter/SourceRewriter.cpp

```diff
@@ -50,6 +50,11 @@ static llvm::cl::opt<bool>
                   llvm::cl::cat(SourceRewriterCategory),
                   llvm::cl::desc("Back up source files before rewriting"));
 
+static llvm::cl::list<std::string>
+    IgnorePrefixes("ignore-prefix",
```

Comment:
Might not be necessary since only things under `--root-directory` are rewritten.

Reply:
It's not necessary, and in fact is unused now. We could probably just drop this.